src/util.rs

```diff
@@ -312,6 +313,90 @@ pub async fn unzip_file(
     Ok(ret)
 }
 
+pub async fn zip_files(
+    files: Vec<impl AsRef<Path>>,
```

Comment:
if you don't need ownership of the path objects at all, consider using a slice:
`&[impl AsRef<Path>>]`